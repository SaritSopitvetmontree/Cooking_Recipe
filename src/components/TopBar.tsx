import React from 'react';
import { AppBar, Toolbar, Typography } from '@mui/material';

const TopBar: React.FC = () => {
  return (
    <AppBar position="static">
      <Toolbar>
        <Typography variant="h6">Cooking App</Typography>
      </Toolbar>
    </AppBar>
  );
};

export default TopBar;
