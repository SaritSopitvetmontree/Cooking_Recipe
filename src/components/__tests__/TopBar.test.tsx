import { render } from '@testing-library/react';
import TopBar from '../TopBar';

it('renders homepage unchanged', () => {
  const { container } = render(<TopBar />)
  expect(container).toMatchSnapshot()
})