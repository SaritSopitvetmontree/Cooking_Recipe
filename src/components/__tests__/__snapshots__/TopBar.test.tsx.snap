// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation0 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-1s4j34q-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
    >
      <h6
        class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
      >
        COOKING-APP
      </h6>
    </div>
  </header>
</div>
`;
